doc/md/privacy.md

```diff
@@ -449,15 +454,14 @@ func Do(ctx context.Context, client *ent.Client) error {
 ```
 
 We finish our example with another privacy-rule named `DenyMismatchedTenants` on the `Group` schema.
-The `DenyMismatchedTenants` rule rejects the group creation if the associated users don't belong to
+The `DenyMismatchedTenants` rule rejects group creation if the associated users don't belong to
 the same tenant as the group.
 
-```go
-// DenyMismatchedTenants is a rule runs only on create operations, and returns a deny decision
-// if the operation tries to add users to groups that are not in the same tenant.
+```go title="examples/privacytenant/rule/rule.go"
+// DenyMismatchedTenants is a rule that returns a deny decision if the operation
+// involves users that don't belong to the same tenant as the group or users that are from more than one tenant.
```

Comment:
```suggestion
```go title="examples/privacytenant/rule/rule.go"
// DenyMismatchedTenants is a rule that runs only on create operations and returns a deny
// decision if the operation tries to add users to groups that are not in the same tenant.
```